integration/spark-common/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala

```diff
@@ -232,38 +230,6 @@ object CarbonReflectionUtils {
     (ctor.newInstance(conArgs: _*), clazz)
   }
 
-  /**
-   * It is a hack to update the carbon input, output format information to #HiveSerDe
-   */
-  def updateCarbonSerdeInfo(): Unit = {
-    val currentMirror = scala.reflect.runtime.currentMirror
```

Comment:
I think this was done for presto scenario. Spark writes a carbon table to hive metastore. That time serde was not updated. so the read from presto was failing. It may not work now. Can check in cluster

Reply:
added back